mkdocs/themes/mkdocs/base.html

```diff
@@ -65,16 +69,16 @@
     </head>
 
     <body{% if page and page.is_homepage %} class="homepage"{% endif %}>
-        <div class="navbar fixed-top navbar-expand-lg navbar-{% if config.theme.nav_style == "light" %}light{% else %}dark{% endif %} bg-{{ config.theme.nav_style }}">
-            <div class="container">
+        <div class="navbar fixed-top navbar-expand-lg bg-{{ config.theme.nav_style }}" data-bs-theme="{% if config.theme.nav_style == 'light' %}light{% else %}dark{% endif %}">
+            <div class="container-fluid">
```

Comment:
I'm noticing that this makes the navigation take up the whole page width. I think it's better to keep it like it was - aligned with the page content. More in line with the original idea of the theme. **Also** Bootstrap already has a wider page width anyway.

```suggestion
            <div class="container">
```

Reply:
Applied this suggestion - reverted this in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae